lib/kube/proxy/forwarder.go

```diff
@@ -2614,7 +2614,22 @@ func (f *Forwarder) removeKubeDetails(name string) {
 // KubeProxy services or remote clusters are automatically forwarded to
 // the final destination.
 func (f *Forwarder) isLocalKubeCluster(sess *clusterSession) bool {
-	return !sess.authContext.teleportCluster.isRemote && f.cfg.KubeServiceType == KubeService
```

Comment:
This change affects not only tests, correct? Was this a bug or how otherwise it affects actual Kube Access?

Reply:
It affects kube access only if you use the legacy Kubernetes proxy mode that (almost) no one uses. Kube proxy + kube service works correctly.

I caught it during the execution (tests were already helpful)